tests/std/tests/GH_002030_asan_annotate_string/test.cpp

```diff
@@ -0,0 +1,1825 @@
+            assert(verify_string(assign_iterator));
+        }
+
+#if 0 // TRANSITION, DevCom-1527920
+        str assign_iterator_sso{input_sso};
+        try {
+            assert(verify_string(assign_iterator_sso));
+            assign_iterator_sso.assign(begin(iter_data), end(iter_data));
+        } catch (...) {
+            assert(verify_string(assign_iterator_sso));
+        }
+#endif
+
+        str assign_input_iterator{input};
+        try {
+            assert(verify_string(assign_input_iterator));
+            assign_input_iterator.assign(input_iter_data.begin(), input_iter_data.end());
+            assert(false);
+        } catch (...) {
+            assert(verify_string(assign_input_iterator));
+        }
+
+#if 0 // TRANSITION, DevCom-1527920
+        str assign_input_iterator_sso{input_sso};
+        try {
+            assert(verify_string(assign_input_iterator_sso));
+            assign_input_iterator_sso.assign(input_iter_data.begin(), input_iter_data.end());
+            assert(false);
+        } catch (...) {
+            assert(verify_string(assign_input_iterator_sso));
+        }
+#endif
+    }
+
+    { // insert
+        str insert_iterator{input};
+        try {
+            assert(verify_string(insert_iterator));
+            insert_iterator.insert(insert_iterator.begin(), begin(iter_data), end(iter_data));
+            assert(false);
+        } catch (...) {
+            assert(verify_string(insert_iterator));
+        }
+
+#if 0 // TRANSITION, DevCom-1527920
+        str insert_iterator_sso{input_sso};
+        try {
+            assert(verify_string(insert_iterator_sso));
+            insert_iterator_sso.insert(insert_iterator_sso.begin(), begin(iter_data), end(iter_data));
+            assert(false);
+        } catch (...) {
+            assert(verify_string(insert_iterator_sso));
+        }
+#endif
+
+        str insert_input_iterator{input};
+        try {
+            assert(verify_string(insert_input_iterator));
+            insert_input_iterator.insert(insert_input_iterator.begin(), input_iter_data.begin(), input_iter_data.end());
+            assert(false);
+        } catch (...) {
+            assert(verify_string(insert_input_iterator));
+        }
+
+#if 0 // TRANSITION, DevCom-1527920
+        str insert_input_iterator_sso{input_sso};
+        try {
+            assert(verify_string(insert_input_iterator_sso));
+            insert_input_iterator_sso.insert(
+                insert_input_iterator_sso.begin(), input_iter_data.begin(), input_iter_data.end());
+            assert(false);
+        } catch (...) {
+            assert(verify_string(insert_input_iterator_sso));
+        }
+#endif
+    }
+}
+
+template <class Alloc>
+void run_tests() {
+    test_construction<Alloc>();
+    test_append<Alloc>();
+    test_assign<Alloc>();
+    test_insertion<Alloc>();
+    test_removal<Alloc>();
+    test_misc<Alloc>();
+    test_sstream<Alloc>();
+#if !defined(__clang__) && !defined(_M_) // TRANSITION, LLVM-54804

```

Comment:
is this `_M_` stray? We don't define it anywhere.

Reply:
Why yes, that should be _M_64